src/tests/state/router/selectors.test.js

```diff
@@ -0,0 +1,156 @@
+import { LOCATION_CHANGE } from 'connected-react-router/lib/actions';
+import { fromJS } from 'immutable';
+import * as matchers from 'jest-immutable-matchers';
```

Comment:
These aren't generic matchers, but matchers specific to Immutable. Can this be something like `import * as immutableMatchers from 'jest-immutable-matchers';`?